analytic_partner_hr_timesheet/models/account_analytic_line.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Antiun Ingeniería S.L. - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, models
+
+
+class AnalyticAccountLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    @api.onchange('project_id')
+    def on_change_project_id(self):
+        if self.project_id:
```

Comment:
Because in V10, in the timesheet activities list view, it is project_id instead of account_id as it was in V8:

![detaledactivities](https://user-images.githubusercontent.com/5930419/27276895-df6e72c0-54dc-11e7-81ea-1d4898d49b2f.png)